packages/teams-js/test/public/app.spec.ts

```diff
@@ -578,7 +604,7 @@ describe('AppSDK-app', () => {
         error: 'Something went wrong...',
```

Comment:
>'Something went wrong...' [](http://example.com/codeflow?start=15&length=25)

I know you didn't introduce this, but can you please use a variable for this message? We use it several times in this file.